modules/blocks/test/registration.js

```diff
@@ -2,50 +2,57 @@
  * External dependencies
  */
 import { expect } from 'chai';
+import sinon from 'sinon';
 
 /**
  * Internal dependencies
  */
 import * as blocks from '../registration';
 
+/* eslint-disable no-console */
```

Comment:
As a convention can we put these on the very first line followed by a newline? Just so that it's immediately obvious the rule doesn't apply to the file.